node/src/components/block_validator/keyed_counter.rs

```diff
@@ -43,7 +39,7 @@ where
                 1
             }
             Some(value) => {
-                value.add_assign(1);
```

Comment:
What is the reason for this and the `sub_assign` change? Not an issue, just curious.

Reply:
Just to be "normal" and also reduce the imports required :)